frontend/src/metabase/query_builder/components/expressions/ExpressionEditorHelpText.styled.tsx

```diff
@@ -1,38 +1,71 @@
 import styled from "@emotion/styled";
 import { monospaceFontFamily } from "metabase/styled-components/theme";
+import ExternalLink from "metabase/core/components/ExternalLink";
+import Icon from "metabase/components/Icon";
 import { color } from "metabase/lib/colors";
 
 export const Container = styled.div`
-  padding: 1.5rem;
+  padding: 1.25rem 1rem 1.25rem;
 
-  font-size: 0.875rem;
+  font-size: 14px;
+  line-height: 20px;
 `;
 
 export const FunctionHelpCode = styled.div`
-  padding: 0.75rem 1rem 0.75rem;
-  margin: 0.5rem 0 1.5rem;
-
-  background-color: ${color("bg-light")};
-
   color: ${color("text-dark")};
   font-family: ${monospaceFontFamily};
 `;
 
 export const FunctionHelpCodeArgument = styled.span`
-  color: ${color("brand")};
+  color: ${color("accent3")};
 `;
 
-export const ExampleBlock = styled.div`
-  color: ${color("text-light")};
+export const Divider = styled.div`
+  height: 1px;
+  background-color: ${color("border")};
+  margin: 1.25rem -1rem 1rem -1rem;
 `;
 
-export const ExampleTitleText = styled.div`
-  margin-bottom: 0.25rem;
+export const ArgumentsGrid = styled.div`
+  margin: 1rem 0;
 
-  font-size: 0.75rem;
-  font-weight: 700;
+  display: grid;
+  gap: 1rem;
+  grid-template-columns: repeat(2, minmax(2rem, max-content));
+`;
+
+export const ArgumentItem = styled.div`
+  color: ${color("accent3")};
+
+  font-size: 12px;
```

Comment:
Please change `font-size` and `line-height` to rem

Reply:
is it a requirement? @ranquild

Reply:
I believe we almost always use `rem` for font and space